.github/workflows/apiTests.yml

```diff
@@ -0,0 +1,22 @@
+name: API-tests
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
```

Comment:
**suggestion**: you can add here support for `.nvmrc` file

https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file